src/test/java/mpack/Assignement2.java

```diff
@@ -0,0 +1,59 @@
+package mpack;
+
+import java.time.Duration;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+//import org.openqa.selenium.WindowType;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class Assignement2 {
+	public static void main(String[] args)
+	{
+		WebDriverManager.chromedriver().setup();
```

Comment:
If possible move this to base class and call it from there instead of writing it in each class .This will reduce common /duplicate code

Reply:
Duplicate comment